Makefile

```
@@ -43,7 +43,7 @@ build-docs-website: dev-docs
 	cd docs && npm run build
 	cp -R docs/public/* dist/
 
-docs-dev:
+docs-local:
 	cd docs && npm run start
 
 docs-api-dev:
```

Should we change the API docs too? It's another local web server run by `pdoc3` 

```suggestion
docs-api-local:
```